### PR TITLE
Fix Badge Overlapping And Icon

### LINKED ISSUE #1950: Fix Overlapping Badge & Footer
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behaviour:**

#1926  Created bug of overlapping Badges. As Size of Badge Preview is required to be large enough so , we need to keep height of footer as small as possible.
**Expected behaviour:**

Fixing Overlapping Badges.
**Steps to reproduce:**

**Screenshots**

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1950

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

Now 
ezgif com-video-to-gif 9

### CODE REVIEW

File: frontend/app/components/create-badge.js [hunk 1]
```diff
@@ -10,16 +10,13 @@ export default Component.extend({
...
           card[0].style.position = 'fixed';
           card[0].style.right = '0';
           card[0].style.top = '0';
-          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '600px';
-        } else if (e.type === 'scroll' && this.window.innerWidth >= 764 && (EmberObject.$(window).scrollTop() + EmberObject.$(window).height() > (EmberObject.$(document).height() - 100))) {
-          card[0].style.position = 'fixed';
           EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '530px';
         } else {
-          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '100%';
+          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '530px';
```
Comment (rajvaibhavdubey) [inline frontend/app/components/create-badge.js]: These changes are not required.

Comment (sk9331657) [reply frontend/app/components/create-badge.js]: There is no effect of removing this code.

**Comment by sk9331657** [comment]
@mariobehling  Please review this pr . This will fix footer and increase preiew area.

**Comment by mariobehling** [comment]
Please don't overlap the footer over the badge.

**Comment by sk9331657** [comment]
> The idea would be that as soon as the footer is close to the badge, that the badge just moves up as well.

Ok I will do that way.

**Comment by sk9331657** [comment]
@mariobehling  For time being this pr can fix overlapping . I will fix the remaining bug in other pr.

**Comment by mariobehling** [comment]
@sk9331657 Would be great, if you could finalize this PR @abishekvashok pointed out. Thank you.

**Comment by sk9331657** [comment]
> @sk9331657 Would be great, if you could finalize this PR @abishekvashok pointed out. Thank you.

Will surely fix this. Busy in making Google Summer of Code proposal right now.

**Comment by sk9331657** [comment]
Screenshot from 2019-04-12 22-39-23
Screenshot from 2019-04-12 22-39-29

@abishekvashok  @mariobehling @gabru-md @yashLadha @ParthS007   Finalised the pr. :smile:

File: frontend/app/templates/components/footer-main.hbs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{!-- <footer>
```
Comment (abishekvashok) [inline frontend/app/templates/components/footer-main.hbs]: Why was the specification removed?

Comment (sk9331657) [reply frontend/app/templates/components/footer-main.hbs]: @abishekvashok  This specification comments the code. I have just reverted the commented code with adding other features and scroll able footer.

**Comment by sk9331657** [comment]
> @sk9331657 The bug was to remove the overlapping, which can also be solved without changing the design of footer. Changing the design if footer is not required.

@rajvaibhavdubey  Increasing the height of footer limits the size of badge and restrict size of badge details which is not desired.

**Comment by sk9331657** [comment]
@abishekvashok  @gabru-md  Please review.

**Comment by rajvaibhavdubey** [comment]
Closing due to inactivity.

**Pull request closed without merging**